main_test.go

```diff
@@ -31,259 +32,260 @@ func Test_runVerify(t *testing.T) {
 		{
 			name:     "valid main branch default",
 			artifact: "./testdata/binary-linux-amd64-workflow_dispatch",
-			source:   "github.com/asraa/slsa-on-github-test",
+			source:   "github.com/laurentsimon/slsa-verifier-test-gen",
```

Comment:
Was changing this necessary for some reason?

Reply:
I see. We need to re-generate them by building in a github action but I guess doing it in this repo is problematic? Maybe a workflow that is only triggered by `workflow_dispatch` would work?

We probably want to have a readme in the `testdata` directory to document how to regenerate it.

Reply:
Agreed on all these steps. What repo name shall we create?

Reply:
Under ossf-tests in fine probably